dv/uvm/icache/data/common_sim_cfg.hjson

```diff
@@ -0,0 +1,128 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  project:          opentitan
+  doc_server:       docs.opentitan.org
+  results_server:   reports.opentitan.org
+
+  flow:             sim
+  flow_makefile:    "{proj_root}/dv/uvm/icache/data/sim.mk"
+
+  import_cfgs:      ["{proj_root}/dv/uvm/icache/data/common_project_cfg.hjson",
+                     "{proj_root}/dv/uvm/icache/data/common_modes.hjson",
+                     "{proj_root}/dv/uvm/icache/data/fusesoc.hjson",
+                     "{proj_root}/dv/uvm/icache/data/{tool}/{tool}.hjson"]
+
+  // Default directory structure for the output
+  build_dir:          "{scratch_path}/{build_mode}"
+  run_dir_name:       "{index}.{test}"
+  run_dir:            "{scratch_path}/{run_dir_name}/out"
+  sw_build_dir:       ""
+  sw_root_dir:        ""
+
+  // pass and fail patterns
+  pass_patterns:    ["^TEST PASSED (UVM_)?CHECKS$"]
+  fail_patterns:    ["^UVM_ERROR\\s[^:].*$",
+                     "^Assert failed: ",
+                     "^\\s*Offending '.*'",
+                     "^TEST FAILED (UVM_)?CHECKS$"]
+
```

Comment:
Reminder to update this as needed for Ibex.

Reply:
Yep, those modifications will be coming in a later PR if needed.